datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/prometheus.py

```diff
@@ -53,24 +52,34 @@ def parse_metrics(endpoint):
 def get_options_text(options):
     return '\n{}\n' 'q - Quit'.format('\n'.join('{} - {}'.format(n, option) for n, option in enumerate(options, 1)))
 
+def set_up_stream(ctx, endpoint, filehandle):
+    if endpoint and filehandle:
+        ctx.fail("Please specify an endpoint or a file but not both.")
+    if endpoint:
+        return requests.get(sanitize_endpoint(endpoint), stream=True).iter_lines(decode_unicode=True)
+    elif filehandle:
+        return filehandle
+    else:
+        ctx.fail("Please specify an endpoint or a file.")
```

Comment:
very nit comment, but since you are not using elif line 58:

```suggestion
    if endpoint:
        return requests.get(sanitize_endpoint(endpoint), stream=True).iter_lines(decode_unicode=True)
    if filehandle:
        return filehandle
    ctx.fail("Please specify an endpoint or a file.")
```